src/emails/index.ts

```diff
@@ -1,13 +1,17 @@
 import creationCompteCandidat from './candidats/creationCompteCandidat';
-import invitationActiveCompte from './invitationCreateAccount/invitationActiveCompte';
+import invitationActiveCompte from './invitation/invitationActiveCompte';
```

Comment:
ok c'est du pinaillage : 
```suggestion
import invitationActiveCompte from './invitations/invitationActiveCompte';
```